config/controller/kustomization.yaml

```diff
@@ -6,4 +6,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: public.ecr.aws/aws-controllers-k8s/rds-controller
-  newTag: v0.1.0
+  newTag: v0.0.0-non-release-version
```

Comment:
uh-oh... this shouldn't have happened... @brucegucode what version of code-generator did you use?

Reply:
Might also need to run `git fetch --all --tags`

Reply:
Ah, i see, need to fetch the latest tags

Reply:
updated in rev2 with correct tag